app/src/main/java/com/dcrandroid/util/Utils.kt

```diff
@@ -217,6 +223,77 @@ object Utils {
         manager.notify(Constants.TRANSACTION_SUMMARY_ID, groupSummary)
     }
 
+    fun registerProposalNotificationChannel(context: Context) {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+            val channel = NotificationChannel(Constants.PROPOSAL_CHANNEL_ID, context.getString(R.string.app_name), NotificationManager.IMPORTANCE_DEFAULT)
+            channel.enableLights(true)
+            channel.enableVibration(true)
+            channel.lockscreenVisibility = Notification.VISIBILITY_PUBLIC
+            channel.importance = NotificationManager.IMPORTANCE_LOW
+
+            val notificationManager = context.getSystemService(Context.NOTIFICATION_SERVICE) as NotificationManager
+            notificationManager.createNotificationChannel(channel)
+        }
+    }
+
+    fun sendProposalNotification(context: Context, manager: NotificationManager, proposalID: Long, title: String,
+                                 token: String) {
+
+        val gson = GsonBuilder().registerTypeHierarchyAdapter(ArrayList::class.java, Deserializer.ProposalDeserializer()).create()
+        val proposalResult = multiWallet!!.politeia!!.getProposalByID(proposalID)
```

Comment:
```suggestion
        val proposalResult = WalletData.multiWallet!!.politeia!!.getProposalByID(proposalID)
```

Reply:
adding `WalletData` comes up as redundant

Reply:
It's redundant because you added `multiWallet` as a static import which is what I'm trying to avoid. Add `WalletData` and remove the `multiWallet` import.

Reply:
okay, i understand now, thanks